lightning/src/chain/onchaintx.rs

```diff
@@ -170,6 +170,18 @@ impl Writeable for Option<Vec<Option<(usize, Signature)>>> {
 	}
 }
 
+#[cfg(anchors)]
+pub(crate) enum ExternalHTLCClaim {
+	/// The claim commonly referred to as the pre-signed second-stage HTLC transaction.
+	SecondStage {
+		amount: u64,
```

Comment:
This could be precised `amounst_sats` to be sure we're talking about satoshis, not msat.